src/components/RichTextEditor/plugins/linkPlugin/FloatingLink/CustomFloatingLink.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button, ButtonSize, ButtonStyle } from '@components/Button/Button';
+import { LINK_CLASSES } from '@components/RichTextEditor/components';
+import { IconCheckMark, IconPen, IconTrashBin } from '@foundation/Icon';
+import {
+    floatingLinkActions,
+    submitFloatingLink,
+    useEditorRef,
+    useFloatingLinkSelectors,
+    useFloatingLinkUrlInput,
+} from '@udecode/plate';
+import React, { useState } from 'react';
+import { FloatingLink } from './FloatingLink';
+
+export const CustomFloatingLink = () => {
+    const isEditing = useFloatingLinkSelectors().isEditing();
+    const editor = useEditorRef();
+    const [isUrlInValid, setIsUrlInValid] = useState(false);
+    const urlHtmlProps = useFloatingLinkUrlInput({});
+
+    const input = (
+        <div data-test-id="floating-link-insert" className="tw-bg-white tw-rounded tw-shadow tw-p-7 tw-min-w-[400px]">
+            <FloatingLink.TextInput />
+            <div className="tw-pt-5">
+                <FloatingLink.UrlInput />
+            </div>
+            {isUrlInValid && <div className="tw-text-red-65 tw-mt-3">Url is invalid.</div>}
```

Comment:
Is it now just on `confirm` possible to check if the url is valid? I was just wondering, as the old behaviour is that the confirm button is disabled. 
If we keep the new way I would adapt the error msg to something like `Please enter a valid Url.`

Reply:
fair :D. I also think it's not important atm as we don't have mobile support yet.